packages/app/src/app/pages/Sandbox/Editor/Workspace/items/NotOwnedSandboxInfo/NotOwnedSandboxInfo.tsx

```diff
@@ -21,14 +26,17 @@ export const NotOwnedSandboxInfo = () => {
       >
         <Files setEditActions={setEditActions} />
       </WorkspaceItem>
-
       <WorkspaceItem
         defaultOpen
         style={{ marginTop: '.5rem' }}
         title="Dependencies"
       >
         <Dependencies />
       </WorkspaceItem>
+      {console.log(sandbox)}
```

Comment:
@SaraVieira Don't forget to  delete this `console.log` 😉

Reply:
🤦🏻‍♀️Deleted